api-lib/event_triggers/addReplyInteractionEvent.ts

```diff
@@ -0,0 +1,50 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
```

Comment:
@sshmm can we rename this and other new ones to match the other InteractionEvent namings weve done so far, for example `createContributionInteractionEvent` so this should be `createReply...`